sdk/src/Core/Amazon.Runtime/Internal/Util/ValueStringBuilder.cs

```diff
@@ -0,0 +1,309 @@
+// Partially adapted from https://raw.githubusercontent.com/dotnet/runtime/main/src/libraries/Common/src/System/Text/ValueStringBuilder.cs
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+#nullable enable
+
+namespace AWSSDK.Core.NetStandard.Amazon.Runtime.Internal.Util
```

Comment:
Since this is source coming from the `dotnet/runtime` repo lets move this to a folder called `RuntimeBackports` under the [ThirdParty](https://github.com/aws/aws-sdk-net/tree/main/sdk/src/Core/ThirdParty) namespace of core. The file's namespace should be `ThirdParty.RuntimeBackports`. I'll figure out the attribution.